client/src/pages/ProfileListing/ProfileList.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC } from 'react';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
```

Comment:
Again - we're using v4 of MUI here. We need to use v5 of MUI. Can we just get rid of that package so we aren't running into that issue?